tests/api/admin/controller/test_controller.py

```diff
@@ -92,9 +92,9 @@ def test_redirect_to_sign_in(self, admin_ctrl_fixture: AdminControllerFixture):
             assert 302 == response.status_code
             location = response.headers.get("Location")
             assert "sign_in" in location
-            assert "admin%2Fweb" in location
-            assert "collection%2Fa%252F(b)" in location
-            assert "book%2Fc%252F(d)" in location
+            assert "admin/web" in location
+            assert "collection/a%252F(b)" in location
+            assert "book/c%252F(d)" in location
```

Comment:
I can't quite figure out where this changed in werkzeug, but the characters as part of GET parameters are not escaped anymore in an `url_for`.
Except for `&` and `=`